packages/replay-next/components/console/renderers/MessageRenderer.tsx

```diff
@@ -97,12 +99,32 @@ function MessageRenderer({
   const primaryContent =
     argumentValues.length > 0 ? (
       <Suspense fallback={<Loader />}>
-        {argumentValues.map((argumentValue: ProtocolValue, index: number) => (
-          <Fragment key={index}>
+        {argumentValues.map((argumentValue: ProtocolValue, index: number) => {
+          const argumentInspector = (
             <Inspector context="console" pauseId={message.pauseId} protocolValue={argumentValue} />
```

Comment:
Doesn't `Inspector` render object types with `<Expandable>` around them anyway?

Is the issue here that it _also_ fetches the preview and we don't need that in this case? Does this error have some kind of a different structure than our standard objects?

Reply:
> Doesn't Inspector render object types with <Expandable> around them anyway?

No, for Error objects it doesn't (since they're not expandable).